### DISCUSSION TITLE
How i can use criterion on Windows?

### DISCUSSION THREAD

**Discussion by es23m022** [Snaipe/Criterion#523] [Q&A]
How i can use criterion on Windows? I can't find a cmake to build the library. Or what can i do to get a static library?

**Comment by lordmichael95** [comment]
Hello [es23m022](https://github.com/es23m022), I think this [link](https://earthly.dev/blog/installandrun-cmake-on-windows/) will help you out on installing cmake for Windows. This [link](https://cmake.org/cmake/help/book/mastering-cmake/chapter/Getting%20Started.html) is a direct link to the people who develop cmake in case you want to learn more about cmake.

Hopefully that helps! If so, feel free to mark this post as the answer.
